agent/agent_test.go

```diff
@@ -32,6 +32,16 @@ func TestAgentClose(t *testing.T) {
 	}
 }
 
+func TestUseCustomConfigDir(t *testing.T) {
```

Comment:
Let's instead use @rakyll's suggestion of an environment variable and move this back into internal, making this test:
```go
key := "GOPS_CONFIG_DIR"
oldDir := os.Getenv(key)
defer os.Setenv(key, oldDir)

newDir := "foo-bar"
os.Setenv(key, newDir)
configDir, err := ConfigDir()
if err != nil {
    t.Fatal(err)
}
if g, w := configDir, newDir; g != w {
    t.Errorf("ConfigDir: got=%q want=%q", g, w)
}
```